daffodil-udf/src/test/java/org/badudfs/annotations/StringFunctions/StringFunctionsProvider.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.badudfs.annotations.StringFunctions;
+
+import org.apache.daffodil.udf.UserDefinedFunction;
+import org.apache.daffodil.udf.UserDefinedFunctionProvider;
+
+/**
+ * UDF Provider for Negative Unit test
+ *
+ * Contains classes with invalid or missing annotations
+ */
+public class StringFunctionsProvider extends UserDefinedFunctionProvider {
+	@Override
+	public UserDefinedFunction lookupInitializedUserDefinedFunction (String namespace, String name) {
```

Comment:
Any specific reason to have this function implemented now that we have the one from the UseDefinedFunctionProvider?

Reply:
Since this is a UDF to test bad annotations, it made sense to me for the bad actor to display the bad case in an equally bad switch. Should I nix for the default instead?

Reply:
Makes sense to me.